src/components/features/playlist/PlaylistControls.tsx

```diff
@@ -85,17 +85,23 @@ const PlaylistSelectItem = forwardRef<HTMLDivElement, PlaylistSelectItemProps>(
                 <Group noWrap>
                     <Stack spacing={0} w="100%">
                         <Text size="sm">{label}</Text>
-                        {/* @ts-ignore */}
-                        <Text color={others["data-selected"] ? colors.gray[5] : colors.gray[6]} size="xs">
+                        <Text
```

Comment:
For the record, this is the only code update in this PR -- to work around the Prettier reformatting breaking a `@ts-ignore`. This code update removes the `@ts-ignore` by making TypeScript happy.